File: csharp/Platform.Data.Doublets.Sequences.Tests/ILinksExtensionsTests.cs
using Xunit;

namespace Platform.Data.Doublets.Sequences.Tests
{
    /// <summary>
    /// <para>
    /// Represents the links extensions tests.
    /// </para>
    /// <para></para>
    /// </summary>
    public class ILinksExtensionsTests
    {
        /// <summary>
        /// <para>
        /// Tests that format test.
        /// </para>
        /// <para></para>
        /// </summary>
        [Fact]
        public void FormatTest()
        {
            using (var scope = new TempLinksTestScope())
            {
                var links = scope.Links;
                var link = links.Create();
                var linkString = links.Format(link);
                Assert.Equal("(1: 1 1)", linkString);
            }
        }
    }
}

File: cpp/Platform.Data.Doublets.Sequences.Tests/ILinksExtensionsTests.cpp
namespace Platform::Data::Doublets::Sequences::Tests
{
    TEST_CLASS(ILinksExtensionsTests)
    {
        public: TEST_METHOD(FormatTest)
        {
            using (auto scope = TempLinksTestScope())
            {
                auto links = scope.Links;
                auto link = links.Create();
                auto linkString = links.Format(link);
                Assert::AreEqual("(1: 1 1)", linkString);
            }
        }
    };
}
